### ISSUE TITLE
Column "ProcessedOn" missing in table A_Asset_Split

### ISSUE THREAD

**Issue by marcalwestf** [adempiere/adempiere#1890]
When running the Application Server, an error is generated.
The command line shows it as follows:
```
19:33:06,607 ERROR [STDERR] ===========> AcctProcessor.postSession: 
SELECT * FROM A_Asset_Split WHERE AD_Client_ID=? 
AND (ProcessedOn=?) AND Processed='Y' AND Posted='N' AND IsActive='Y' 
ORDER BY Created [44]
org.postgresql.util.PSQLException: ERROR: column "processedon" does not exist

```
The column is neither existent on the Application Dictionary nor in the database.

**Comment by yamelsenih** [comment]
Hi @marcalwestf it was introduced in #1656 issue

**Comment by PraponAon** [comment]
to solve this problem, I create column.

ALTER TABLE A_Asset_Split ADD COLUMN processedon numeric;

**Comment by e-Evolution** [comment]
solved #1656
